docker/service/src/main/java/uk/nhs/adaptors/scr/fhirmappings/ConditionMapper.java

```diff
@@ -0,0 +1,123 @@
+package uk.nhs.adaptors.scr.fhirmappings;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Condition;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Period;
+
+import uk.nhs.adaptors.scr.exceptions.FhirMappingException;
+import uk.nhs.adaptors.scr.models.GpSummary;
+import uk.nhs.adaptors.scr.models.gpsummarymodels.AllConditions;
+import uk.nhs.adaptors.scr.models.gpsummarymodels.ConditionObject;
+import uk.nhs.adaptors.scr.models.gpsummarymodels.EndTime;
+import uk.nhs.adaptors.scr.utils.DateUtil;
+
+public class ConditionMapper {
+    public static void mapConditions(GpSummary gpSummary, List<Condition> conditions) throws FhirMappingException {
+        AllConditions conditionListObject = new AllConditions();
+        List<AllConditions> conditionParent = new ArrayList<>();
+
+        List<ConditionObject> conditionObjectList = new ArrayList<>();
```

Comment:
Optionally this can be changed into java streams:
```
List<ConditionObject> conditionObjectList = conditions.stream()
    .map(this:: mapCondition)
    .collect(Collectors.toList());
```
which imho is more readable

Reply:
Should we change this and also change the other mapping classes to be non static?

Reply:
How's that related to my comment?